pilot/pkg/model/jwks_resolver.go

```diff
@@ -296,12 +299,11 @@ func (r *JwksResolver) BuildLocalJwks(jwksURI, jwtIssuer, jwtPubKey string) *env
 }
 
 // CreateFakeJwks is a helper function to make a fake jwks when istiod failed to fetch it.
-func CreateFakeJwks(jwksURI string) string {
-	// Create a fake jwksURI
-	fakeJwksURI := "Error-IstiodFailedToFetchJwksUri-" + jwksURI
-	// Encode jwksURI with base64 to make dynamic n in jwks
-	encodedString := base64.RawURLEncoding.EncodeToString([]byte(fakeJwksURI))
-	return fmt.Sprintf(`{"keys":[ {"e":"AQAB","kid":"abc","kty":"RSA","n":"%s"}]}`, encodedString)
+func CreateFakeJwks() string {
```

Comment:
Is this a const, can we cache it

Reply:
updated

Reply:
This is still not stable and will result in XDS draining as we hop between instances every 30min. This is really bad and this exact issue has caused massive production issues for our customers in the past./

This must be static and deterministic.